### DISCUSSION TITLE
Custom Addon to flag texts based on their description/explanation.

### DISCUSSION THREAD

**Discussion by mxx-lxg** [WeblateOrg/weblate#11299] [Q&A]
Hello everyone,
I'm would like to write an addon that will flag all new source texts based on their description contents, but i'm having a hard time finding what i need in the documentation.
We are using React-intl and the idea is that our developers can add flags to message descriptions so that the respective texts can automatically be flagged when source files are being pulled from the repo.
Is there any way to do this in a custom addon with calling external scripts?

**Comment by nijel** [comment]
https://docs.weblate.org/en/latest/admin/addons.html#bulk-edit might be able to do that if the description is loaded into Weblate.

**Reply by mxx-lxg** [reply]
This seems promising! I'll check it out. Thanks.
